Formula/ntp.rb

```diff
@@ -22,20 +22,25 @@ class Ntp < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "16137579ea8978ef71a1222071cd9e44a42eeeaa8d83ed83bdbf0fd048e6178c"
   end
 
-  depends_on "openssl@1.1"
+  depends_on "openssl@3"
+
+  # Patch for glibc 2.34+ compatibility
+  patch do
+    on_linux do
+      url "https://raw.githubusercontent.com/Homebrew/formula-patches/6c508ba/ntp/ntp-4.2.8p15-glibc.patch"
+      sha256 "f41a8066ce41f3f1d066801f54ead54b66d13e71355a50c20836d303f8b300e4"
+    end
+  end
```

Comment:
Is this upstreamed?

Reply:
I did not find a place to submit this patch or see this patch shown up in the upstream.

In fact, according to https://lore.kernel.org/all/20210923233409.50cba2f7@gmx.net/T/

 >> Yeah, it looks like ntp is a pretty dead-end project...

 > Maybe time to provide NTPsec [1] as alternative...

 > Regards,
 > Peter

 > [1] https://www.ntpsec.org/

Reply:
Maybe we can simply deprecate the formulae in this same PR?